### DISCUSSION TITLE
move asdf-gctx here please

### DISCUSSION THREAD

**Discussion by chiefy** [asdf-community/.github#22] [Ideas]
Just made this but would like to have it here?

- https://github.com/chiefy/asdf-gctx

**Comment by smorimoto** [comment]
Consult this discussion post: https://github.com/orgs/asdf-community/discussions/57 This is the smoothest way to go. Of course I can do all the work for you, but it's done on a best effort basis. (I've been busy with work lately...)

**Reply by chiefy** [reply]
will do, thanks
